tslint.json

```diff
@@ -11,6 +11,7 @@
         "indent": [true,
             "spaces"
         ],
+        "inline-comment-spacing": true,

```

Comment:
These are our own internal rules. Some of them might be suitable to submit to tslint itself, if you want to.